TauAnalysis/MCEmbeddingTools/python/EmbeddingPythia8Hadronizer_cfi.py

```diff
@@ -0,0 +1,44 @@
+import FWCore.ParameterSet.Config as cms
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.Pythia8CUEP8M1Settings_cfi import *
+from GeneratorInterface.ExternalDecays.TauolaSettings_cff import *
+
+
+
+generator = cms.EDFilter("Pythia8HadronizerFilter",
+  
+  maxEventsToPrint = cms.untracked.int32(1),
+  nAttempts = cms.uint32(1000),
+  HepMCFilter = cms.PSet(
+    filterName = cms.string('EmbeddingHepMCFilter'),
+    filterParameters = cms.PSet(
+            ElElCut = cms.untracked.string('El1.Pt > 22 && El2.Pt > 10'),
+            ElHadCut = cms.untracked.string('El.Pt > 28 && Had.Pt > 25'),
```

Comment:
nominal policy for parameters that can change physics behavior is that they should be tracked.

The EmbeddingHepMCFilter is proposed for Generator/Core, which is for centrally supported code.
So, this implementation better have the parameters tracked.